interfaces/Portal/src/app/program/table-filter-row/table-filter-row.component.html

```diff
@@ -12,13 +12,17 @@
       class="util--flex-row ion-align-items-center"
       style="flex-basis: 80%"
     >
-      <span>
+      <span
+        class="ion-align-items-center ion-text-right"
+        style="width: 9rem"
```

Comment:
I think this should be a `min-width` or a `max-width` only.. Or a percentage.

Because otherwise the layout isn't as flexible as it should be.